README.md

```diff
@@ -27,4 +33,24 @@ You can still use all the commands that work with NestJS, for example:
 
 ```bash
 cuckoo generate|g controller <name>
-```
\ No newline at end of file
+```
+
+### Use our schematics
+
+If you want to use the schematics solely, install the schematics globally
+
+```bash
+npm install -g @guidesmiths/cuckoojs-schematics
+```
+
+Currently we have schematics for:
+- creating a .gitignore file:
+
+```bash
+schematics @guidesmiths/cuckoojs-schematics:commitlint
+```
+- adding commitlint:
+
+```bash
+schematics @guidesmiths/cuckoojs-schematics:gitignore
+```
```

Comment:
Swapped ;)

Reply:
Maybe also adding the `directory` option?

```bash
schematics @guidesmiths/cuckoojs-schematics:gitignore --directory=<path>
```